api/src/lib/knex.js

```diff
@@ -1,7 +1,7 @@
 var config = require('../../config');
 var knex = require('knex')({
   debug : process.env.DEBUG,
-  client : 'pg',
+  client : config('/dbAdapter') === 'postgres' ? 'pg' : config('/dbAdapter'),
```

Comment:
Hmm, yeah. It was working though. Knex must accept `postgresql` as an alias for `pg`. Will update to just use `/dbAdapter`